packages/@expo/config-plugins/src/utils/Updates.ts

```diff
@@ -59,9 +60,12 @@ export function getNativeVersion(
  * Compute runtime version policies.
  * @return an expoConfig with only string valued platform specific runtime versions.
  */
-export const withRuntimeVersion: (config: ExpoConfig) => ExpoConfig = (config) => {
+export const withRuntimeVersionAsync: (config: ExpoConfig) => Promise<ExpoConfig> = async (
```

Comment:
here is an async plugin that may not work as expected. i searched the code base and it seems the `withRuntimeVersion` is not being used. maybe we could just remove the whole plugin. especially that i'm not sure to mutate app.config here would be a good idea or not.